spring-cloud-config-server/pom.xml

```diff
@@ -42,6 +42,15 @@
 			<artifactId>spring-boot-starter-actuator</artifactId>
 			<optional>true</optional>
 		</dependency>
+		<dependency>
+			<groupId>io.micrometer</groupId>
+			<artifactId>micrometer-observation</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.aspectj</groupId>
+			<artifactId>aspectjrt</artifactId>
+			<optional>true</optional>
```

Comment:
This causes all the tests in the samples module to fail with org.aspectj.lang.ProceedingJoinPoint

Reply:
We could but we would need to apply changes in A LOT of places (but it could be done). I'll take another look at this